### PR TITLE
Add adr for server side session

### PR DESCRIPTION
**Pull request by JulianRoesner**
# Short Description
This adds the adr for the use of server side sessions

# Feedback
- Do you think sth is missing?

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: doc/adr/0002-use-server-side-sessions-with-redis.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Accepted
+
+## Context
+
+So far the session data of the user is stored in a cookie using Flask session. The cookie has a size limit of 4 kb and therefore this limits the size of the data in the session. To be independent of the cookie size, we want to use server side session data.
+
+## Decision
+
+We use a server side session stored that are stored in a redis database.
```
Comment (punknoir101) [inline doc/adr/0002-use-server-side-sessions-with-redis.md:15]: _that are stored_  can be removed

**Review by punknoir101 — COMMENTED** [review]
Maybe we should also add a comment why we go the way with redis client -could be confusing for devs who have already worked with session redis in other frameworks.

**Comment by JulianRoesner** [comment]
> Maybe we should also add a comment why we go the way with redis client -could be confusing for devs who have already worked with session redis in other frameworks.

Good idea. I can not find anything in the TechSpec regarding this point. @SannyNguyenHung ?

**Comment by nfelger** [comment]
@JulianRoesner reminder to link the tech spec as discussed in dev weekly

**Review by SannyNguyenHung — APPROVED** [review]
LGTM :)

**Pull request merged**